packages/library-base/src/commonMain/kotlin/io/realm/internal/InternalRealmConfiguration.kt

```diff
@@ -33,4 +33,13 @@ interface InternalRealmConfiguration : RealmConfiguration {
     val nativeConfig: NativePointer
     val notificationDispatcher: CoroutineDispatcher
     val writeDispatcher: CoroutineDispatcher
+
+    fun debug(): String {
```

Comment:
```suggestion
    override fun toString(): String {
```

Reply:
This is not possible on an interface. "An interface may not implement a method of 'Any'"